test/cmd/image-lookup.sh

```diff
@@ -15,6 +15,9 @@ project="$( oc project -q )"
 os::test::junit::declare_suite_start "cmd/image-lookup"
 # This test validates image lookup resolution
 
+# TODO: fix and re-enable these tests before 4.0 release
+exit 0
```

Comment:
don't `exit` here, maybe just comment out

`exit` in non-top-level things could do weird stuff with the control flow IIRC